sys/kern/src/task.rs

```diff
@@ -108,7 +108,7 @@ impl Task {
             unsafe { Ok(slice.assume_readable()) }
         } else {
             Err(FaultInfo::MemoryAccess {
-                address: Some(slice.base_addr() as u32),
+                address: Some(slice.base_addr() as usize),
```

Comment:
`slice.base_addr()` is already a `usize`.

Reply:
Fixed in next patchset.